pkgs/data/icons/antu-icon-theme/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, stdenvNoCC, fetchFromGitLab, gtk3 }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "antu-icon-theme";
+  version = "ff16444b";
```

Comment:
the version should be unstable-$date_of_last_upstream_commit if you use the rev

Reply:
I believe guidelines were updated recently, and unstable without previous version tags should be prefixed with `0-` (i.e. `0-unstable-YYYY-MM-DD`): https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming

Reply:
uh til, thanks ^^

Reply:
Like this?
`version = "0-unstable-2024-01-23";
`

`rev = "279be277";`